test/npcomp-run-mlir/invalid-conv_2d_nchw.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: not npcomp-run-mlir %s \
+// RUN:   -invoke conv_2d_nchw \
+// RUN:   -arg-value="dense<[[[[1.0, 0.0], [1.0, 1.0]]]]> : tensor<1x1x2x2xf32>" \
+// RUN:   -arg-value="dense<[[[[1.0, 2.0], [3.0, 4.0]], [[1.0, 2.0], [3.0, 4.0]]]]> : tensor<1x2x2x2xf32>" \
+// RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
+// RUN:   | FileCheck %s --check-prefix=CHANNELS
+
+// RUN: not npcomp-run-mlir %s \
+// RUN:   -invoke conv_2d_nchw \
+// RUN:   -arg-value="dense<[[[[1.0, 0.0], [1.0, 1.0]]]]> : tensor<1x1x2x2xf32>" \
+// RUN:   -arg-value="dense<[[[[1.0, 2.0], [3.0, 4.0]]], [[[1.0, 2.0], [3.0, 4.0]]]]> : tensor<2x1x2x2xf32>" \
+// RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
+// RUN:   | FileCheck %s --check-prefix=CHANNELSF
+
+// RUN: not npcomp-run-mlir %s \
+// RUN:   -invoke conv_2d_nchw \
+// RUN:   -arg-value="dense<[[[[1.0, 0.0], [1.0, 1.0]]]]> : tensor<1x1x2x2xf32>" \
+// RUN:   -arg-value="dense<[[[[1.0, 2.0], [3.0, 4.0], [5.0, 6.0]]]]> : tensor<1x1x3x2xf32>" \
+// RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
+// RUN:   | FileCheck %s --check-prefix=HEIGHT
+
+// RUN: not npcomp-run-mlir %s \
+// RUN:   -invoke conv_2d_nchw \
+// RUN:   -arg-value="dense<[[[[1.0, 0.0], [1.0, 1.0]]]]> : tensor<1x1x2x2xf32>" \
```

Comment:
for the purpose of this test, you can just use `dense<0.0> : tensor<...>`, since we don't care about the tensor contents.

Reply:
OK. I didn't pay attention to the tensor contents since it's strictly a test on shapes, but I'll apply your suggestion.

Reply:
For the most part, I just copied-and-pasted the `invalid-matmul.mlir` test.

Reply:
Oops, I should have followed my own advice there ;) Thanks for catching that!